kubernetes/deployment-production.tmpl

```diff
@@ -0,0 +1,225 @@
+    spec:
+      containers:
+        - name: panoptes-production-app
+          image: zooniverse/panoptes:__IMAGE_TAG__
+          env:
+            - name: RAILS_ENV
+              value: production
+            - name: SECRET_KEY_BASE
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-production-env-vars
+                  key: SECRET_KEY_BASE
+            - name: ATTENTION_REDIS_URL
+              value: redis://panoptes-redis.zooniverse.org:6379/1
+            - name: AWS_REGION
+              value: us-east-1
+            - name: KINESIS_STREAM
+              value: zooniverse-production
+            - name: ZOO_STREAM_KINESIS_STREAM_NAME
+              value: zooniverse-production
+            - name: ZOO_STREAM_SOURCE
+              value: panoptes
+            - name: SUBJECTS_BUCKET
+              value: zooniverse-static
+            - name: BUCKET_PATH
+              value: panoptes-uploads.zooniverse.org
+            - name: EXPORT_BUCKET_PATH
+              value: panoptes-exports.zooniverse.org
+            - name: AWS_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-common-env-vars
+                  key: AWS_ACCESS_KEY_ID
+            - name: AWS_SECRET_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: panoptes-common-env-vars
+                  key: AWS_SECRET_ACCESS_KEY
+            - name: DUMP_CONGESTION_OPTS_INTERVAL
+              value: 86400
+            - name: DUMP_CONGESTION_OPTS_MAX_IN_INTERVAL
+              value: 1
+            - name: DUMP_CONGESTION_OPTS_MIN_DELAY
+              value: 43200
+            - name: DUMP_CONGESTION_OPTS_REJECT_WITH
+              value: cancel
+            - name: COUNTER_CONGESTION_OPTS_INTERVAL
+              value: 360
+            - name: COUNTER_CONGESTION_OPTS_MAX_IN_INTERVAL
+              value: 10
+            - name: COUNTER_CONGESTION_OPTS_MIN_DELAY
+              value: 180
+            - name: COUNTER_CONGESTION_OPTS_REJECT_WITH
+              value: cancel
+            - name: CORS_ORIGIN_USERS
+              value: /^https?:\/\/((www|staging)\.antislaverymanuscripts\.org|(www|staging)?\.?heirtagger\.ox\.ac\.uk|(www|lab|learn)\.wildcamgorongosa\.org|www\.diagnosislondon\.org|anno\.tate\.org\.uk|(www|relaunch|field-book)\.notesfromnature\.org|[a-z0-9-]+\.oldweather\.org|www\.scribesofthecairogeniza\.org|([a-z0-9-]+\.)?zooniverse\.org|[a-z0-9-]+\.(pfe-)?preview\.zooniverse\.org)(:\d+)?$/
+            - name: CORS_ORIGIN_OAUTH
+              value: /^https?:\/\/((www|staging)\.antislaverymanuscripts\.org|(www|staging)?\.?heirtagger\.ox\.ac\.uk|(www|lab|learn)\.wildcamgorongosa\.org|www\.diagnosislondon\.org|anno\.tate\.org\.uk|(www|relaunch|field-book)\.notesfromnature\.org|[a-z0-9-]+\.oldweather\.org|www\.scribesofthecairogeniza\.org|([a-z0-9-]+\.)?zooniverse\.org|[a-z0-9-]+\.(pfe-)?preview\.zooniverse\.org)(:\d+)?$/
```

Comment:
does this inject as a regex from k8s? I think it'll be a string, right?

Reply:
Are these two variables actually used? I don't see any references to them:

```
17:20 $ git grep CORS_ORIGIN_OAUTH
kubernetes/deployment-production.tmpl:            - name: CORS_ORIGIN_OAUTH
kubernetes/deployment-staging.tmpl:            - name: CORS_ORIGIN_OAUTH
✔ ~/Code/Panoptes [kubeify {origin/kubeify}|✔] 
17:22 $ git grep CORS_ORIGIN_USERS
kubernetes/deployment-production.tmpl:            - name: CORS_ORIGIN_USERS
kubernetes/deployment-staging.tmpl:            - name: CORS_ORIGIN_USERS
```

Reply:
The variables were renamed. I've updated the K8 configs to match.